src/contact.rs

```diff
@@ -95,6 +95,29 @@ impl ContactId {
         self.0
     }
 
+    /// Sets display name for existing contact.
+    ///
+    /// Display name may be an empty string,
```

Comment:
Shouldn't we put all documentation to the top-level (JSON-RPC) API, or how are users supposed to know such details?